src/utils/dom.ts

```diff
@@ -46,7 +46,8 @@ export async function withIframe<T>(
       // This code checks for the loading state manually.
       // See https://github.com/fingerprintjs/fingerprintjs/issues/645
       const checkReadyState = () => {
-        if (iframe.contentWindow?.document.readyState === 'complete') {
+        // Make sure iframe.contentWindow and iframe.contentWindow.document are both loaded
```

Comment:
According to your error stack trace, this situation can happen in [JSDOM](https://github.com/jsdom/jsdom). Could you please edit the comment so that it says that the `contentWindow.document` can miss in JSDOM.